src/index.html

```diff
@@ -52,26 +52,46 @@ <h3>Your Current Team:</h3>
             </li>
           </ul>
         </div>
-        <div class="journal">
-          <h3>Yesterdays Journal:</h3>
+        <h3>Yesterdays Journal:</h3>
+        <div class="journal dashboard-journal">
           <h6>Rating:</h6>
-          <img src="./Star.svg" alt="star icon" class="star-gap" />
-          <img src="./Star.svg" alt="star icon" class="star-gap" />
-          <img src="./Star.svg" alt="star icon" class="star-gap" />
-          <img src="./Star.svg" alt="star icon" class="star-grey star-gap" />
-          <img src="./Star.svg" alt="star icon" class="star-grey star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img
+            src="./svgs/Star.svg"
+            alt="star icon"
+            class="star-grey star-gap"
+          />
+          <img
+            src="./svgs/Star.svg"
+            alt="star icon"
+            class="star-grey star-gap"
+          />
 
           <h6>Comprehension:</h6>
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" />
-          <img src="./Rectangle.svg" alt="rectangle icon" class="icon-grey" />
-          <img src="./Rectangle.svg" alt="rectangle icon" class="icon-grey" />
-          <img src="./Rectangle.svg" alt="rectangle icon" class="icon-grey" />
```

Comment:
we wrote in these greyed out boxes.. the icon grey ones?? we dont need them anymore?

Reply:
they are all still there, just replaced the path ;)